src/berrybrew.cs

```diff
@@ -181,7 +221,7 @@ internal static void Config()
 
         internal static string Version()
         {
-            return "0.12.1.20160302";
+            return "0.12.1.2016040101";
```

Comment:
Ok, that sounds reasonable. I didn't quite understand what you were doing
there at first.

On Mon, Apr 4, 2016 at 2:23 PM, hraharahra <notifications@github.com> wrote:

> In src/berrybrew.cs
> <https://github.com/dnmfarrell/berrybrew/pull/17#discussion_r58441947>:
>
> > @@ -181,7 +221,7 @@ internal static void Config()
> >
> >          internal static string Version()
> >          {
> > -            return "0.12.1.20160302";
> > +            return "0.12.1.2016040101";
>
> After I take more deeper look to the changes, I will change it to
> 0.12.2.20160400. Last two digits are for iterations, not day of month. For
> 100 iterations per month are *more than plenty*. 100 of them per day are
> way, way excessive. Date string is to ease correlation with latest
> Strawberry releases.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/dnmfarrell/berrybrew/pull/17/files/b9fb4ce94f417c1c897ceb7d83f80c4bbcfb9bc8#r58441947>
>